inst/workflow/1_mappings/ENROLL.yaml

```diff
@@ -0,0 +1,28 @@
+meta:
+  Type: Mapped
+  ID: ENROLL
+  Description: Enrollment Data Mapping 
+  Priority: 1
+spec: 
+  Raw_ENROLL:
+    studyid:
+      required: true
+      type: character
+    invid:
+      required: true
+      type: character
+    country:
+      required: true
+      type: character
+    subjectid:
+      required: yes
```

Comment:
@lauramaxwell super minor but did we want to change this stiill?

Reply:
oh. yeah- i'd like to standardize those. i'll replace these with true in a new branch